### PR TITLE
Remove check for resourceVersion

### PR DESCRIPTION
**Pull request by apoorva11029**
This check was added earlier as we were seeing Ansible tasks run repeatedly, that's not happening anymore.

### CODE REVIEW

File: roles/accprovision/tasks/main.yml [hunk 1]
```diff
@@ -1,61 +1,63 @@
...
-    path: /tmp/acc_provision/version
-    state: touch
-  when: not file_present.stat.exists
+- name: Write cmap YAML
+  copy:
+    dest: "{{ acc_provision_dir_path }}/cmap.yaml"
+    content: "{{ cmap_content }}"
 
-- name: Set old resourceVersion to empty string if no cached value exists
+- name: Set input YAML
```
Comment (abhis2112) [inline roles/accprovision/tasks/main.yml:24]: Collecting config-map Spec

File: roles/accprovision/tasks/main.yml [hunk 2]
```diff
@@ -1,61 +1,63 @@
...
-- name: Get current resourceVersion if version file exists
+- name: Get acc-provision configmap
+  k8s_info:
+    api_version: v1
+    kind: ConfigMap
+    name: acc-provision-config
+    namespace: "{{ lookup('env', 'WATCH_NAMESPACE') }}"
+  register: cmap_data
+
+- name: Set fact
```
Comment (abhis2112) [inline roles/accprovision/tasks/main.yml:15]: Collecting config-map content

Comment (apoorva11029) [reply roles/accprovision/tasks/main.yml]: thanks, changed

File: roles/accprovision/tasks/main.yml [hunk 3]
```diff
@@ -1,61 +1,63 @@
...
+    chdir: "{{ acc_provision_dir_path }}"
+
+- name: Run k8s command to apply deployment file
+  k8s:
+    state: present
+    src: "{{ acc_provision_dir_path }}/{{ aci_cni_deployment_file }}"
+  register: k8s_result
+  until: k8s_result is not failed
+  retries: 5
+  delay: 30
```
Comment (abhis2112) [inline roles/accprovision/tasks/main.yml:63]: is this reconcile also has to be 60 seconds with 5 retries similar to reconcilePeriod in watches.yaml. Or is 30 sec is sufficient?

**Review by abhis2112 — COMMENTED** [review]
Request review

File: roles/accprovision/tasks/main.yml (hunk 3 above)
Comment (apoorva11029) [reply roles/accprovision/tasks/main.yml:63]: This is just to retry this k8s call- the reconcilePeriod is to run the entire Ansible role every 60s.

Comment (abhis2112) [reply roles/accprovision/tasks/main.yml:63]: So this timeout is sufficient [Tested]. Just confirming

**Pull request merged**